remote/process_test.go

```diff
@@ -177,7 +177,7 @@ func addBasicProcessTests(tests ...processTestCase) []processTestCase {
 				opts := testutil.SleepCreateOpts(3)
 				proc, err := makep(ctx, opts)
 				require.NoError(t, err)
-				assert.Error(t, proc.RegisterSignalTriggerID(ctx, jasper.SignalTriggerID(-1)))
```

Comment:
pre go1.15 this was Kosher?

Reply:
Yes. An int is treated as a char, which is then converted to a string.